### PR TITLE
Export more Typescript types from Select components

### PR DESCRIPTION
**Pull request by CarlosCortizasCT**
#### Summary

Export more Typescript types from Select components

## Description

In order to help consumers using `SelectInput` and `SelectField` components, I propose to expose more TS types from them.

### CODE REVIEW

File: packages/components/fields/select-field/src/select-field.tsx [hunk 1]
```diff
@@ -196,7 +196,7 @@ export type TSelectFieldProps = {
    * <br/>
    * The value will be the selected option, or an array of options in case isMulti is true.
    */
-  onChange?: (event?: TCustomEvent) => void;
+  onChange?: (event: TCustomEvent) => void;
```
Comment (CarlosCortizasCT) [inline packages/components/fields/select-field/src/select-field.tsx:199]: The event was marked as optional but this callback is just passed internally to the `SelectInput` component where the event is not optional.

**Review by kark — APPROVED** [review]
Thank you 🙌🏅
Just a kind reminder about a changeset.

**Review by Rhotimee — APPROVED** [review]
Nice work. Just changeset missing. Thanks

**Comment by CarlosCortizasCT** [comment]
Changesets added here: 43221be4d61c0d510ed2d9a2b6e38adf1c468f35

Thanks for the reminder 🙇

**Pull request merged**